lib/MusicBrainz/Server/Controller/Work.pm

```diff
@@ -102,7 +102,7 @@ with 'MusicBrainz::Server::Controller::Role::Merge' => {
     edit_type => $EDIT_WORK_MERGE,
 };
 
-before qw( show aliases tags details edit ) => sub {
+before qw( show aliases tags details edit edit_annotation open_edits edits ) => sub {
```

Comment:
I ordered this top to bottom on sidebar order since the others seem to be order LTR according to tab order.

Reply:
Are the languages and work attributes needed on these new pages? If not, probably better to add a separate `before` that just loads the type.

Reply:
Is this better?